lms/static/build.js

```diff
@@ -18,7 +18,15 @@
          * done.
          */
         modules: getModulesList([
-            'teams/js/teams_tab_factory'
+            'js/edxnotes/views/page_factory',
```

Comment:
@peter-fogg Only include 'js/student_account/views/finish_auth_factory' in this list (in addition to teams_tab_factory, which exists on the branch your PR is based off of), as the others are all added by individual PRs.